[IndexTable] Only run sticky logic if we have a defined rootBounds object (#11584)
Shopify/polaris: .changeset/moody-worms-search.md

---
'@shopify/polaris': patch
---

> Updated logic in use-is-select-all-actions-sticky logic to not set any sticky behaviour if we do not have access to the root element

Reviewer: ```suggestion
Updated `useIsSelectAllActionsSticky` logic to not set any sticky behaviour if we do not have access to the root element
```
